drivers/sensor/lis2dh/lis2dh.h

```diff
@@ -114,6 +114,11 @@
 #define LIS2DH_FS_SELECT(fs)		((fs) << LIS2DH_FS_SHIFT)
 #define LIS2DH_FS_BITS			(LIS2DH_FS_SELECT(LIS2DH_FS_IDX))
 #define LIS2DH_ACCEL_SCALE(range_g)	((SENSOR_G * 2 * (range_g)) / 65636LL)
+#if defined(CONFIG_LIS2DH_OPER_MODE_HIGH_RES)
+	#define LIS2DH_HR_BIT		BIT(3)
+#else
+	#define LIS2DH_HR_BIT		BIT(0)
```

Comment:
Maybe
#define LIS2DH_HR_BIT		0

BIT(0) is 1
Isn't it?

Reply:
Yes, you are right. I will fix that.